project/src/main/scala/org/broadinstitute/clio/publish/ArtifactoryFatJarPublishingPlugin.scala

```diff
@@ -49,13 +51,7 @@ object ArtifactoryFatJarPublishingPlugin extends AutoPlugin {
       Seq(
         publishTo := Some(ivyArtifactoryResolver),
         publishMavenStyle := false,
-        isSnapshot := true,
-        moduleSettings := InlineConfigurationWithExcludes(
-          projectID.value,
-          projectInfo.value,
-          Seq.empty,
-          configurations = Seq(Default)
-        )
```

Comment:
I think we need to keep the `moduleSettings` so gradle can find the jar in the zamboni nightly test.

Reply:
yeah, I couldn't get it to compile so I removed it for now, but I assumed some stuff will be broken until it's added back in